java/src/jmri/jmrix/loconet/LnPortController.java

```diff
@@ -53,6 +53,8 @@ public boolean okToSend() {
     protected boolean mInterrogateAtStart = true;
     protected boolean mTranspondingAvailable = false;
 
+    protected boolean mLoconetProtocolAutoDetect = true;
```

Comment:
Maybe it's good to have this default to false in case the method `setLoconetProtocolAutoDetect()` isn't called. But no problem if you prefer it as is.